magento/magento2
# [Catalog] Fixing the Products grid with default values on multi stores

### Description (*)
This PR fixes the products grid to use the default values on multi storing website.

### Fixed Issues (if relevant)

1. magento/magento2#13338: Products grid in admin does not display default values?
2. ...

### Manual testing scenarios (*)
1. Make sure you have a product
2. Make sure you have several store views
3. Set a different value for Product Name for each store view

For more details, check task's steps.

### Contribution checklist (*)
